dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/user/hibernate/HibernateUserStore.java

```diff
@@ -619,4 +619,20 @@ public void setActiveLinkedAccounts(
       update(user);
     }
   }
+
+  @Override
+  public User getUserByVerificationToken(String token) {
+    Query<User> query =
+        getSession().createQuery("from User u where u.verifiedEmail like :token", User.class);
+    query.setParameter("token", token + "%");
+    return query.uniqueResult();
+  }
+
+  @Override
+  public User getVerifiedEmail(String email) {
+    Query<User> query =
+        getSession().createQuery("from User u where u.verifiedEmail like :email", User.class);
```

Comment:
is there a reason why `like` is being used here over `=`?
do we prevent using `%` in an email?